internal/app/api/api.go

```diff
@@ -113,7 +122,9 @@ func Init(conns *InfraConns) (http.Handler, error) {
 
 	visitLogger := logger.WithField("package", "visit")
 	visitSrv := visit.NewService(visitRepo, cusVisitAssocRepo, actVisitAssocRepo, visitLogger)
-	visitHandler := visithandler.New(visitSrv, visitLogger)
+	emailLogger := logger.WithField("package", "email")
+	emailService := email.NewSendMailService(visitRepo, customerRepo, cusVisitAssocRepo, emailLogger)
```

Comment:
I have not look at the code for a long time until now.
So what is the purpose of initializing logger and inject it to `NewSendMailService` like this?
Except for custom logging when testing which is not so important, I don't see any benefits of doing this.

Reply:
Because the base of project been made before. so i thing, When we inject the custom logging to a service can help us manage logger easily